etc/server.properties

```diff
@@ -46,9 +46,8 @@ redis.port=6379
 # Access control
 access.group.admin=unc:app:lib:cdr:admin
 
-# Java Message Server (localhost, 61616)
-jms.host=localhost
-jms.port=:61617
+# Java Message Server
+jms.brokerUrl=tcp://localhost:61617
```

Comment:
should this be 61617 or 61616? i'm just not clear why the old comment had a different port listed